src/main/resources/application.conf

```diff
@@ -32,27 +18,36 @@ spray.can {
   }
 }
 
+workflow-options {
+  // These workflow options will be encrypted when stored in the database
+  encrypted-fields: ["refresh_token"]
+
+  // AES-256 key to use to encrypt the values in `encrypted-fields`
+  base64-encryption-key: "q78yJ9CtbWTX3+zDHJgHSiN4I7+nq5kgYjOD94nWK48="
```

Comment:
Is application.conf accessible externally? I think Thurloe currently has this in application.conf as well on the assumption that only the service itself is able to reach it.